apps/tailwind-components/assets/css/theme/aumc.css

```diff
@@ -34,5 +34,14 @@
     --text-color-search-filter-group-title: var(--text-color-title);
     --text-color-pagination-input: var(--color-blue-800);
 
-    --border-radius-input: var(--border-radius-50px);
-  }
\ No newline at end of file
+    --border-radius-3px: 3px;
+    --border-radius-50px: 50px;
+    --border-radius-input: 50px;
+    --border-radius-textarea-input: 20px;
+    --border-radius-search-input: 9999px;
```

Comment:
It might be easier to switch to percents here. E.g., 80%, 100%, etc.

Reply:
don't think that works with resizable textareas

Reply:
Ah, yes. Makes sense.